tests/mentionui.js

```diff
@@ -852,6 +852,44 @@ describe( 'MentionUI', () => {
 			} );
 		} );
 
+		describe.only( 'callback function using data from editor', () => {
+			beforeEach( () => {
+				return createClassicTestEditor( {
+					feeds: [
+						{
+							marker: '#',
+							feed() {
+								const items = this.editor.config.get( 'mentionData' );
+								return new Promise( resolve => {
```

Comment:
I think you can simplify this test too. To make it synchronous. Also, a more direct check that the feed function is executed in the context of the editor would be even simpler.

Reply:
The feed is now synchronous. I would suggest keeping the test to the editor config to ensure it remains reachable. AFAIK there's no explicit spy to test the bound context of a function.

Reply:
Personally I'm not a big fun of spies anyway :) I could never memorise all the normal assertions, not to mention sinon's. Hence, what I meant would be a simple `expect( this ).to.equal( editor)` inside the feed callback and returning whatever data (so you can remove the config option at all). Because at this point the test is already completed in terms of checks. We only need to make sure that it tears down well. The config option is noise in this test because this test is not about the `mentionData` config option at all – it's only about the context.

Reply:
I see, thanks for the pointer. I updated the test accordingly.